CONTRIBUTING.md

```diff
@@ -48,15 +51,12 @@ Testing can be accomplished on systems capable of hosting supported container ru
    export GOARCH=amd64
    export DOCKER_DEFAULT_PLATFORM=linux/amd64
    ```
-   
+
+   Then, run from your local machine:
+
     ```bash
-    # Local workstation
-    make build/packages/kubernetes/1.19.3/ubuntu-18.04
-    make build/packages/kubernetes/1.19.3/images
-    make dist/containerd-1.6.8.tar.gz && tar xzvf dist/containerd-1.6.8.tar.gz
-    make dist/weave-2.8.1.tar.gz && tar xzvf dist/weave-2.8.1.tar.gz
-    make dist/openebs-3.3.0.tar.gz && tar xzvf dist/openebs-3.3.0.tar.gz
-    make dist/registry-2.8.1.tar.gz && tar xzvf dist/registry-2.8.1.tar.gz
+    # To build the sample under the [hack/testdata](./hack/testdata) directory targeting Ubuntu 22.04
+    make build/sample/ubuntu-22.04
```

Comment:
I strongly disagree with changing this. The point of this set of commands is not to be something that the average developer runs. The point of this set is to expose, in clear form, exactly what commands need to be run for a build - and let people know how to change versions etc.

Reply:
We still able to change the versions and run all make bundles manually. It is an optional helper. 
Note that without it is required to:
- Copy and past from the latest.sh script the INSTALLER spec OR from a doc
- Run make per make which lead to does not know if you executed all that wish or not
- Looking every time what is that target to build the make command?  

Then note that with the helper you just need to update the sample and the makefile target to do the tests.
It makes the life easier (ihmo) for tests purposes.

Reply:
If you want to have both the 'make all' and this, fine. But 'go edit this makefile' is a different level of discoverability than 'here's the command to build this addon version'.

Reply:
> If you want to have both the 'make all the versions for one spec' command and this, great! 
Hun, are you suggesting a script that read the scripts/Manifest and create all commands? 
If so, I think that it is too much effort for a dev helper and I would still needing to copy and pasting the script/manifest spec content, ensure indentation and etc. 

> But 'go edit this makefile' is a different level of discoverability than 'here's the command to build this addon version'
The dev does not need to edit. However, it is much more sample just update the versions on samples and make to do another test then all steps. it is just a dev helper, if someone prefer can copy and paste the commands and run in the line as well. The idea here is just create a full example to make easier test out/understand how it works.